sample/SampleHost/SampleHost.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="5.3.3" />
     <PackageReference Include="Microsoft.Extensions.Configuration.CommandLine" Version="8.0.0" />
     <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.1" />
+    <PackageReference Include="System.Text.Json" Version="6.0.11" />
```

Comment:
This won't be needed as you have pinned to a higher version in the Host.Storage project, which this references